action.yml

```diff
@@ -45,6 +45,10 @@ inputs:
   version-file:
     description: a versions file (e.g. as used by `asdf`), which defines inputs
     default: ''
+  hexpm-mirrors:
+    description: mirror(s) for hex.pm, one per line
+    default: |
+        https://builds.hex.pm
```

Comment:
I'm not sure if YAML (bizarre as it is) ends up parsed as an array for this case, but we should make the examples (and documentation) use `-` instead of `|` (which is a multi-line string, at least conceptually).

Reply:
Ok, yeah, you're right 😄. I didn't know this was a JavaScript GitHub Action limitation, but apparently it is, even if an odd one.